packages/app/src/actions/index.js

```diff
@@ -100,3 +102,17 @@ export const tDoSaveVisualization = (
         return onError('tDoSaveVisualization', error);
     }
 };
+
+export const tDoDeleteVisualization = () => (dispatch, getState) => {
+    const current = sGetCurrent(getState());
+
+    dispatch(
+        fromSnackbar.acReceivedSnackbarMessage({
+            message: i18n.t(`"${current.name}" successfully deleted.`),
```

Comment:
Perhaps leave favorite name out of the i18n?

Reply:
Oops! right!